docs/reference/commandline/run.md

```diff
@@ -262,6 +262,13 @@ https://docs.docker.com/engine/installation/binaries/#/get-the-linux-binary)),
 you give the container the full access to create and manipulate the host's
 Docker daemon.
 
+On Windows, the paths must be specified using Windows-style semantics. Windows
+containers do not support the target of a volume or bind-mount being pre-
+populated in the container image.
```

Comment:
oh, wait, I see; "pre-populated" as in, contain the data from the host.

Is that going to be supported? What is the use of a bind-mounted directory in that case, that's a big difference in semantics